src/components/Dropdown/index.jsx

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React, { useState, useCallback } from "react";
+import * as _ from "lodash";
+
+type Props = {
+  button: string,
```

Comment:
Rename `button` to something more descriptive like - `selectedValue` - and find matching item based on that value and render it's label - don't render `button` (or `selectedValue` itself).